Development environment setup.md

```diff
@@ -1,10 +1,16 @@
 ## Install basic tools
 
-- [Latest Oracle JDK](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
 - [Android Studio](https://developer.android.com/studio/index.html)
 - Git—`brew install git` ([brew docs](http://brew.sh/))
 - Zeplin—`brew cask install zeplin` ([brew cask docs](https://caskroom.github.io/))
 
+## Setup java JDK
+
+Java JDK comes embedded in android studio and is often more compatible, therefore we don't need to install new JDK. However android studio by default uses systems JDK so project structure needs to be changed in order to use embedded one.
```

Comment:
```suggestion
Java JDK comes embedded in Android Studio and is often more compatible, therefore we don't need to install latest JDK. However, Android Studio by default uses systems JDK so project structure needs to be changed in order to use embedded one.
```

Reply:
I have changed android studio to Android studio, however I don't think new should be changed to latest, because we don't want to only indicate that the latest JDK is the problem, but any JDK different from the Android studios embedded one

Reply:
Agree. But new have the same buzz as the latest 😄 Maybe to say: therefore we don't need to manually download & install JDK

Reply:
Great suggestion. I have corrected the text to match your suggestion